hack/update-copyright.sh

```diff
@@ -22,6 +22,17 @@ CURRENT_PATH=$(dirname "$BASH_SOURCE")
 PROTECT_ROOT=$CURRENT_PATH/..
 VENDOR_PATH=$CURRENT_PATH/../vendor
 BOILERPLATE=$CURRENT_PATH/boilerplate.go.txt
+excluded_files=("tools.go" "ibmblockcsi_controller_test.go")
```

Comment:
why are they excluded? was something changed?

Reply:
there is a problem with the make generate that it fails when I add the copyright to `tools.go` file.
and for the `ibmblockcsi_controller_test.go`, I removed it from the exclude

Reply:
then could you open a ticket for `tools.go` and add its number as a TODO comment in this line?

Reply:
the ticket description should contain all the problem details you have (e.g. errors)

Reply:
does this mean `tools.go` should not start with the header for now?

Reply:
the in the `tools.go` is when the copyright is above [this](https://github.com/IBM/ibm-block-csi-operator/blob/cbf6aa2cb1bfb4a860cf607c6d891f3ea8af8225/tools.go#L1) line, so I can put it manually after the problematic line and keep the exclude, and in that way, the copyright will be in the file but it won't fail the CI

Reply:
sounds good to me

Reply:
I don't see the ticket number, could you point me to it?

Reply:
I thought we agreed to [this](https://github.com/IBM/ibm-block-csi-operator/pull/195#discussion_r693918211) and leave it without a ticket

Reply:
oh, then could you include it as part of CSI-3239?

Reply:
I assume the solution would be to automatically add the copyright only after build tags

Reply:
yes
done